src/docker-images/RestfulAPI/runScheduler.sh

```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
-rm /DLWorkspace/src/utils/config.yaml
```

Comment:
Does `ln` replace the old target?

Reply:
No, there are no such file in image, I think this is used for previous deployment when they tried to debug with local change mount directly into this directory. But we don't do this anymore.